textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/queryrewriter/WordBase.java

```diff
@@ -0,0 +1,35 @@
+package edu.uci.ics.textdb.dataflow.queryrewriter;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.net.URL;
+import java.util.HashSet;
+
+/**
+ * Created by shiladityasen on 4/30/16.
+ * Class to provide simple abstraction of a dictionary of words with constant retrieval time
+ * Used by FuzzyTokenizer to process query
+ */
+public class WordBase
```

Comment:
@sandeepreddy602 I implemented, what you pointed out in 0ed8ec337461a47451ea4883a1e941e735a7f4e4. However, I felt that it will be useful to internally hold both the List and the HashSet because, iterating through the HashSet using the present logic will require conversion to an Array anyway.